pytensor/link/pytorch/linker.py

```diff
@@ -37,6 +37,9 @@ def conversion_func_register(*args, **kwargs):
     def jit_compile(self, fn):
         import torch
 
+        # flag that tend to help our graphs
+        torch._dynamo.config.capture_dynamic_output_shape_ops = True
```

Comment:
Hopefully when #1159 gets merged we can just delete this flag altogether since torch will know these aren't dynamic